tutorials/README.md

```diff
@@ -0,0 +1 @@
+Folder służący do przechowywania materiałów edukacyjnych (własnej produkcji lub nie) mających zapoznać resztę zespołu z wykorzystanymi metodami. Staramy się, aby poradniki były zwięzłę oraz zawierały odnośniki do źródeł lub pomocnicznych materiałów.
```

Comment:
literówka - `zwięzłę`, przecinek po `się`?

Reply:
Literówka poprawiona (,ale w opisie commita też zrobiłem literówkę, żeby było po równo hehe). Co do przecinka, to nie widzę tutaj błędu.